src/WhereConditions/WhereConditionsServiceProvider.php

```diff
@@ -11,8 +11,14 @@
 
 class WhereConditionsServiceProvider extends ServiceProvider
 {
+    public const DEFAULT_HAS_OPERATOR = 'GTE';
```

Comment:
This hardcodes a value that is internal to the `SQLOperator`, which is just one possible implementation of the `Operator` interface.

Reply:
Should I just use the real SQL operator directly, or is there some method to get an enum name of `SQLOperator`, using the operator itself? Or should I just create a function like `default` that just return the name directly?

Reply:
Or maybe create a `GREATER_THAN_EQUALS` const in `SQLOperator`